content/webapp/components/Body/Body.tsx

```diff
@@ -172,13 +173,34 @@ const Body: FunctionComponent<Props> = ({
   contentType,
 }: Props) => {
   const { sliceMachine } = useToggles();
+  const featuredTextFromBody = body.find(
+    (slice, i) =>
+      i === 0 && slice.type === 'text' && slice.weight === 'featured'
+  ) as Slice<'text', prismic.RichTextField>;
+  const featuredTextFromUntransformedBody = untransformedBody.find(
+    (slice, i) =>
+      i === 0 && slice.slice_type === 'text' && slice.slice_label === 'featured'
+  ) as prismic.Slice<'text', { text: prismic.RichTextField }>;
+
   const filteredBody = body
+    .filter(
+      (slice, i) =>
+        !(i === 0 && slice.type === 'text' && slice.weight === 'featured')
+    )
     .filter(slice => !(slice.type === 'picture' && slice.weight === 'featured'))
     // The standfirst is now put into the header
     // and used exclusively by articles / article series
     .filter(slice => slice.type !== 'standfirst');
 
   const filteredUntransformedBody = untransformedBody
+    .filter(
+      (slice, i) =>
+        !(
+          i === 0 &&
+          slice.slice_type === 'text' &&
+          slice.slice_label === 'featured'
+        )
+    )
```

Comment:
Am I right that these are doubled up at the moment because of the slice machine toggle/work? 
A readability suggestion could be that those rules (`i === 0 && slice.type === 'text' && slice.weight === 'featured'` and `i === 0 && slice.slice_type === 'text' && slice.slice_label === 'featured'`) be declared as consts, less repetition and easier to skim?